pkg/tui/interface.go

```diff
@@ -0,0 +1,101 @@
+package tui
+
+import (
+	"fmt"
+	"net/netip"
+	"time"
+
+	"github.com/taoky/ayano/pkg/analyze"
+)
+
+type showMode int
+
+const (
+	TopValues showMode = iota
+	Total
+)
+
+func Tui(a *analyze.Analyzer) {
+	// displayRecord is used to remember latest accesses time by specific IP
+	displayRecord := make(map[netip.Prefix]time.Time)
+	serverFilter := ""
+	mode := TopValues
+	ticker := time.NewTicker(time.Duration(a.Config.RefreshSec) * time.Second)
```

Comment:
Should probably create a struct as a substitute for these function-local variables.